bin/strap.sh

```diff
@@ -367,54 +435,18 @@ if [ -f "$DAPTIV_DOTFILES/script/postbrew" ] && [ -x "$DAPTIV_DOTFILES/script/po
   "$DAPTIV_DOTFILES/script/postbrew" 2>/dev/null
 fi
 
-# Setup dotfiles
-if [ -n "$STRAP_GITHUB_USER" ]; then
-  DOTFILES_REPO="$STRAP_GITHUB_USER/dotfiles"
-  if [ -n "$STRAP_CI" ]; then
-    DOTFILES_REPO="daptiv/dotfiles-template"
-  fi
-
-  DOTFILES_URL="git@github.com:$DOTFILES_REPO"
-  log "Checking for remotes on $DOTFILES_URL"
-  git ls-remote "$DOTFILES_URL"
-
-  if git ls-remote "$DOTFILES_URL" &>/dev/null; then
-    log "Fetching $DOTFILES_REPO from GitHub:"
-    if [ ! -d "$HOME/.dotfiles" ]; then
-      log "Cloning to ~/.dotfiles:"
-      git clone $Q "$DOTFILES_URL" ~/.dotfiles
-    else
-      (
-        cd ~/.dotfiles
-        git pull $Q --rebase --autostash
-      )
+# Run user dotfiles scripts
+if [ -n "$USER_DOTFILES_EXISTS" ]; then
+  pushd ~/.dotfiles > /dev/null
```

Comment:
this should be done inside a ` ( ) ` which eliminates the need for pushd and popd (which might get skipped during an error)